articles/quickstart/native/react-native-expo/interactive.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Add Login to your React Native App
+description: This quickstart demonstrates how to add user login to an React Native application using Auth0.
+seo_alias: react-native
+interactive: true
+files:
+  - files/app-json
+  - files/app
+github:
+  path: 00-Login
+topics: 
+  - quickstarts 
+  - native 
+  - react-native
+---
+
+# Add Login to your React Native App
+
+<!-- markdownlint-disable MD002 MD012 MD041 -->
+
+::: warning
+This SDK is not compatible with "Expo Go" app. It is compatible only with Custom Dev Client and EAS builds
+:::
+
+## Configure Auth0 {{{ data-action=configure }}}
+
+To use Auth0 services, you need to have an application set up in the Auth0 Dashboard. The Auth0 application is where you will configure authentication in your project.
+
+### Configure an application
+
+Use the interactive selector to create a new Auth0 application or select an existing application that represents the project you want to integrate with. Every application in Auth0 is assigned an alphanumeric, unique client ID that your application code will use to call Auth0 APIs through the SDK.
+
+Any settings you configure using this quickstart will automatically update for your Application in the <a href="${manage_url}/#/">Dashboard</a>, which is where you can manage your Applications in the future.
+
+If you would rather explore a complete configuration, you can view a sample application instead.
```

Comment:
I would link "a sample application" to the GitHub repo.

Reply:
Looks like this sentence is something standard we have used in many places. And we can't link "Sample Application" as it refers to the one inside the dashboard AFAIK.

<img width="293" alt="Screenshot 2022-09-09 at 10 33 57" src="https://user-images.githubusercontent.com/15910425/189308148-1e7444a4-96ab-41ec-9a57-11c7b607dd3f.png">